packages/core/src/extension.ts

```diff
@@ -171,22 +171,32 @@ export async function activate(context: vscode.ExtensionContext) {
                 switchToAmazonQCommand.register()
                 installAmazonQExtension.register()
 
-                if (!isExtensionInstalled(VSCODE_EXTENSION_ID.amazonq) && isPreviousQUser()) {
-                    void vscode.window
-                        .showInformationMessage(
-                            'Amazon Q has moved to its own VSCode extension.' +
-                                '\nInstall to work with Amazon Q, a generative AI assistant, with chat and code suggestions.',
-                            'Install',
-                            'Learn More',
-                            'Dismiss'
-                        )
-                        .then(async resp => {
-                            if (resp === 'Learn More') {
-                                void learnMoreAmazonQCommand.execute()
-                            } else if (resp === 'Install') {
-                                void installAmazonQExtension.execute()
-                            }
-                        })
+                if (!isExtensionInstalled(VSCODE_EXTENSION_ID.amazonq)) {
+                    if (isPreviousQUser()) {
+                        await installAmazonQExtension.execute()
```

Comment:
Can this fail for any reason? If so, we should fall back to the dialogue box for manual install.

Reply:
>Can this fail for any reason?

I tried to turn-off wifi but the installation is still successful. Cannot find any case when this fails.